google-cloud-bigtable/lib/google/cloud/bigtable.rb

```diff
@@ -163,6 +163,7 @@ def self.new_with_emulator project_id, emulator_host, timeout
       def self.resolve_credentials given_credentials, scope
         credentials = given_credentials || default_credentials(scope: scope)
         return credentials if credentials.is_a? Google::Auth::Credentials
+        return credentials if credentials.is_a? GRPC::Core::Channel
```

Comment:
I couldn't figure out how to start an insecure connection to Bigtable, via this high-level entry point to the library. The underlying `Service` object supports it, but the point of the test proxy is to use the language-specific SDK idiomatically. If there's an existing way to create an insecure channel that I missed, please let me know.

Reply:
Thanks for this. We'll figure out how to fix the rest of the cases later. There is actually a common pattern in the Ruby clients by which passing the symbol `:this_channel_is_insecure` can be used for credentials when creating an insecure channel, but it looks like that code path is not implemented here either.